dashboard/modules/snapshot/snapshot_head.py

```diff
@@ -97,9 +98,9 @@ async def get_serve_info(self):
         # TODO(architkulkarni): Use _internal_kv_list to get all Serve
         # controllers.  Currently we only get the detached one.  Non-detached
         # ones have name = SERVE_CONTROLLER_NAME + random letters.
-        key = format_key(SERVE_CONTROLLER_NAME, SERVE_SNAPSHOT_KEY)
+        key = get_storage_key(SERVE_CONTROLLER_NAME, SERVE_SNAPSHOT_KEY)
 
-        return json.loads(_internal_kv_get(key, client) or "[]")
+        return cloudpickle.loads(_internal_kv_get(key, client) or "[]")
```

Comment:
why change this to cloudpickle? This needs to be json serializable anyways in order to return it in an HTTP response

Reply:
this is to keep ray serve kvstore only deal with bytes type since json.dumps / loads deal with string, last week we had #17489 that changed it to Union[str, bytes].